server/knowledge_graph/flow.py

```diff
@@ -0,0 +1,215 @@
+
+import json
+from typing import List, Dict
+import os 
+from server.utils.ai_helper import get_llm_client   , llm_call,print_messages
+from langchain.schema import SystemMessage, HumanMessage
+import hashlib
+import psycopg2
+from server.utils.github_helper import GithubService
+from utils.graph_db_helper import Neo4jGraph
+neo4j_graph = Neo4jGraph()
+
+
+class FlowQuery:
+    def __init__(self, query: str):
+        self.query = query
+
+class FlowInference:
+    def __init__(self, project_id: str, directory: str, user_id: str):
+        self.project_id = project_id
+        self.directory = directory
+        self.user_id = user_id
+        self.explain_client = get_llm_client(user_id, "gpt-3.5-turbo-0125")
+        self.setup_database()
+
+    def setup_database(self):
+        conn = psycopg2.connect(os.environ['POSTGRES_SERVER'])
+    
+        cursor = conn.cursor()
+        cursor.execute('''
+            CREATE TABLE IF NOT EXISTS inference (
```

Comment:
Creating a story for removing this, and adding an alembic script for handling this. 
We can do it in some other PR